docs/en/concepts-and-designs/mal.md

```diff
@@ -252,6 +252,45 @@ They extract level relevant labels from metric labels, then informs the meter-sy
  - `processRelation(detect_point_label, [service_label1...], [instance_label1...], source_process_id_label, dest_process_id_label, component_label)` extracts `DetectPoint` labels from first argument, the label value should be `client` or `server`.
    extracts `Service` labels from the first array argument, extracts `Instance` labels from the second array argument, extracts `ProcessID` labels from the fourth and fifth arguments of the source and destination.
 
+## Configuration file
+
+The OAP can load the configuration at bootstrap. If the new configuration is not well-formed, the OAP fails to start up. The files
+are located at `$CLASSPATH/otel-rules`.
+
+The file is written in YAML format, defined by the scheme described below. Brackets indicate that a parameter is optional.
+
+A full example can be found [here](../../../oap-server/server-starter/src/main/resources/otel-rules/oap.yaml)
```

Comment:
I think this is not very accurate. Not only OTEL files are using the MAL engine, but meter analysis is also using this. too.

Reply:
This is an example. What do you expect then?

Reply:
I mean we should list MAL file folders.